src/Breadcrumb/Breadcrumb.tsx

```diff
@@ -3,62 +3,49 @@ import { IconComponentType } from "../Icon/Icon"
 import OperationalContext from "../OperationalContext/OperationalContext"
 import { DefaultProps } from "../types"
 import { darken, isModifiedEvent, isOutsideLink } from "../utils"
-import { OperationalStyleConstants } from "../utils/constants"
 import styled from "../utils/styled"
 
 export interface BreadcrumbProps extends DefaultProps {
   /** Navigation property à la react-router <Link/> */
-  to?: string
+  to: string
   onClick?: (ev?: React.SyntheticEvent<React.ReactNode>) => void
   children?: React.ReactNode
   icon?: IconComponentType
+  /** Optionally moves icon from default position on right, to before the text, including appropriate padding */
+  iconLeft?: boolean
 }
 
-const containerStyles = {
+const Container = styled("a")(({ theme }) => ({
+  color: theme.color.primary,
   label: "breadcrumb",
-  "& svg": {
-    marginLeft: 4,
-  },
-}
-
-const Container = styled("span")(containerStyles)
-
-const ContainerLink = styled("a")(containerStyles, ({ theme }: { theme: OperationalStyleConstants }) => ({
-  "&:link, &:visited": {
-    color: theme.color.primary,
-    borderBottom: "1px solid currentColor",
-    "&:hover": {
-      color: darken(theme.color.primary, 5),
-    },
+  "&:hover": {
+    color: darken(theme.color.primary, 5),
   },
 }))
 
-const Content = styled("span")()
-
-const Breadcrumb: React.SFC<BreadcrumbProps> = ({ to, icon: Icon, onClick, ...props }) => {
-  const ContainerComponent: any = to ? ContainerLink : Container
-  return (
-    <OperationalContext>
-      {ctx => (
-        <ContainerComponent
-          {...props}
-          href={to}
-          onClick={(ev: React.SyntheticEvent<Node>) => {
-            if (onClick) {
-              onClick(ev)
-            }
-            if (!isModifiedEvent(ev) && ctx.pushState && to && !isOutsideLink(to)) {
-              ev.preventDefault()
-              ctx.pushState(to)
-            }
-          }}
-        >
-          <Content>{props.children}</Content>
-          {Icon && <Icon size={12} />}
-        </ContainerComponent>
-      )}
-    </OperationalContext>
-  )
-}
+const Breadcrumb: React.SFC<BreadcrumbProps> = ({ to, icon: Icon, onClick, ...props }) => (
+  <OperationalContext>
+    {ctx => (
+      <Container
+        {...props}
+        tabIndex={0}
```

Comment:
if it is `a` it should be focusable by default, so this is redundant I guess

Reply:
Cypress doesn't know that it's focusable sadly 😢

Reply:
I agree

```suggestion
        tabIndex={0} // for cypress
```

could make sense